examples/dyn_phy.py

```diff
@@ -0,0 +1,86 @@
+import sys
+
+sys.path.append("/usr/local/serialbox/python/")
+import serialbox
+import copy
+import json
+from argparse import ArgumentParser, Namespace
+from datetime import datetime
+from typing import Any, Dict, List
+
+import numpy as np
+import yaml
+from mpi4py import MPI
+
+import fv3core
```

Comment:
I'm confused, is this file supposed to live in the integration repo rather than the physics repo? or is it here to provide a wider context to the physics?